internal/cfgplugins/staticnexthop.go

```diff
@@ -154,6 +170,59 @@ func NextHopGroupConfig(t *testing.T, dut *ondatra.DUTDevice, traffictype string
 
 }
 
+// NextHopGroupConfigScale configures the interface next-hop-group config.
+func NextHopGroupConfigScale(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, encapparams OCEncapsulationParams, ni *oc.NetworkInstance, params StaticNextHopGroupParams) *gnmi.SetBatch {
+	if deviations.NextHopGroupOCUnsupported(dut) {
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			ipPrefix := strings.Join(strings.Split(strings.Split(encapparams.GRETunnelDestinationsStartIP, "/")[0], ".")[:3], ".")
+			buildConfig := func(prefix string, labels []int) string {
+				var b strings.Builder
+				for i := 1; i <= encapparams.Count; i++ {
+					b.WriteString(fmt.Sprintf(`
+nexthop-group %s%d type mpls-over-gre
+ tos 96
+ ttl 64
+ fec hierarchical`, prefix, i))
+					for entry, src := range encapparams.GRETunnelSources {
+						b.WriteString(fmt.Sprintf(`
+ entry  %d push label-stack %d tunnel-destination %s.%d tunnel-source %s`,
+							entry, labels[i-1], ipPrefix, i, src))
+					}
+					b.WriteString("\n!")
+				}
+				return b.String()
+			}
+
+			helpers.GnmiCLIConfig(t, dut, buildConfig("1V4_vlan_3_", encapparams.MPLSStaticLabels))
+			helpers.GnmiCLIConfig(t, dut, buildConfig("1V6_vlan_3_", encapparams.MPLSStaticLabelsForIPv6))
+
+		default:
+			t.Logf("Unsupported vendor %s for native command support for deviation 'next-hop-group config'", dut.Vendor())
+		}
+		return nil
+	} else {
+		ipPrefix := strings.Join(strings.Split(strings.Split(encapparams.GRETunnelDestinationsStartIP, "/")[0], ".")[:3], ".")
```

Comment:
You will have much better validation if  you use `net.ParseCIDR` to validate and extract the prefix.


```suggestion
		n, _, err := net.ParseCIDR(encapparams.GRETunnelDestinationsStartIP)
		if err != nil {
			t.Fatalf("invalid IP address %q provided in GRETunnelDestinationsStartIP - err: %v", err)
		}
		ipPrefix := n.String()
```